### PR TITLE
Greentrip Control

### PR DESCRIPTION
**Pull request by ecvick85**
Updated control branch to only have diary, big data and profile with no suggestion system

### CODE REVIEW

File: config.xml [hunk 1]
```diff
@@ -83,35 +83,5 @@
         <splash density="port-xxxhdpi" src="resources/android/splash/drawable-port-xxxhdpi-screen.png" />
     </platform>
     <icon src="resources/android/icon/drawable-xhdpi-icon.png" />
-    <plugin name="phonegap-plugin-push" spec="~1.9.2">
```
Comment (shankari) [inline config.xml]: why are you removing these plugins from config.xml?

File: package.json [hunk 2]
```diff
@@ -9,66 +9,8 @@
   "devDependencies": {
     "phonegap": "7.1.1"
   },
-  "cordova": {
-    "platforms": [
-      "ios",
-      "android"
-    ],
-    "plugins": {
-      "phonegap-plugin-push": {
```
Comment (shankari) [inline package.json]: and package.json?

Comment (ecvick85) [reply package.json]: I don't know why they were removed. I added them back in the most recent commit, and it still works while running locally.

**Comment by shankari** [comment]
one final change - remove `npm-debug.log`

**Comment by shankari** [comment]
Deployed this to a new channel with the same connection settings as the main greentrip channel since I have no evidence that the server has been reconfigured (@valin1? @b-i-l-l-c-a-o?)

https://e-mission.eecs.berkeley.edu/#/client_setup?new_client=greentripcontrol&clear_usercache=true&clear_local_storage=true

@ecvick85 can you test out the deployed version?

**Pull request merged**